opal/runtime/opal_params_core.c

```diff
@@ -211,6 +214,21 @@ int opal_register_util_params(void)
         return ret;
     }
 
+    /* Basic color options: 30=black, 31=red, 32=green,
+     * 33=yellow, 34=blue, 35=magenta, 36=cyan, 37=white
+     * https://en.wikipedia.org/wiki/ANSI_escape_code#Colors */
+    opal_var_dump_colors = "name=34:value=32:valid_values=36";
+    ret = mca_base_var_register("opal", "opal", NULL, "var_dump_colors",
```

Comment:
How would users obtain the list of valid keys ?

Reply:
The default value contains all keys, but indeed, let's write them in the description?